### DISCUSSION TITLE
Welcome to Font Smoothing Adjuster Discussions!

### DISCUSSION THREAD

**Discussion by abyrne10** [bouncetechnologies/Font-Smoothing-Adjuster#13] [General]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by Lilin2015** [comment]
Thank you, I can't believe this works so significant for me.
For about half a year, I blamed my eye strain on M3Pro to PWM. I almost tried everything, including visiting a doctor for cure. Nothing really works, until I turn off the font smoothing.
My problem disappears in one second!
